cmd/recorder/main.go

```diff
@@ -192,9 +192,14 @@ func doRecord(ctx context.Context, c client.Client, gvks []schema.GroupVersionKi
 	return nil
 }
 
-func forEach(c client.Client, gvk schema.GroupVersionKind, listOptions *client.ListOptions, fn func(unstructured.Unstructured) error) error {
+func forEach(ctx context.Context, c client.Client, gvk schema.GroupVersionKind, listOptions *client.ListOptions, fn func(unstructured.Unstructured) error) error {
+	logger := klog.FromContext(ctx)
 	for ok := true; ok; ok = listOptions.Continue != "" {
-		if _, ok := opk8s.IgnoredCRDList[gvkToCRDName(gvk)]; ok {
+		crdName := gvkToCRDName(gvk)
+		if _, ok := opk8s.IgnoredCRDList[crdName]; ok {
+			logger.Error(fmt.Errorf("unexpected CRD %s", crdName),
+				fmt.Sprintf("Orphaned CRD %s found, you should delete "+
+					"it using 'kubectl delete crd %s'", crdName, crdName))
```

Comment:
two nits:
1. go log does not use capitalized message. "Orphaned" --> "orphaned" 
2. the logger is a structured message. so it is better to be
```go
logger.Error( <error>, "please run `kubectl delete crd <crd_name>` to delete the orphaned CRD", "crd", crdName)
```

Reply:
Done.